Feature contact form

eduardosancho opened the pull request:
#What was done in this PR:

Add social media on the footer, which redirects to my profiles.
Add a "contact me" section 📫  and a footer 🦶 :

It has three required fields the collect information, which is then submitted to Formspree.

eduardosancho committed:
Remove style from project two technologies

eduardosancho committed:
Remove more style from project two technologies

Yothu committed:
Add test file

Yothu committed:
Remove padding technologies

eduardosancho committed:
Organize technologies in a Lines

-Using float left on list elements

Yothu committed:
Add margins

Yothu committed:
Add contact me form section skeleton

eduardosancho committed:
Add contact section title and description

eduardosancho committed:
Add form skeleton

eduardosancho committed:
Add name label and input

Yothu committed:
finished form skeleton

eduardosancho committed:
Add style to form text areas

eduardosancho committed:
Download images for footer

eduardosancho committed:
Add style to footer and fix font families

eduardosancho committed:
Add limits and requirements to contact form

eduardosancho committed:
Add formspree to contact form

eduardosancho committed:
Add required to textarea

eduardosancho committed:
Fix linter errors

eduardosancho committed:
Fix label style and form description style

eduardosancho committed:
Add links to social media on footer

eduardosancho committed:
Add link to my resume on the footer

eduardosancho committed:
Add linkedin profile to the resume link in footer

eduardosancho committed:
Fix linters, add hint-report to .gitignore

Gambit142 commented on index.html:
```diff
@@ -216,6 +216,59 @@ <h1>Skills</h1>
             </div>
         </section>
+        <section class="contact-me">
+            <img id="contact-background" src="./images/mobile_geometry_1.svg" alt="background-box">
+            <div class="contact-me-form">
```
Wonderful work implementing most of the requirements. Your project is great, but it can be better.

- [x] Kindly change this to a `background-image`. If an image is a design, it is best to use it as a `background-image` instead of an actual image. It is a rule of thumb to use an actual image `(img)` if it has relevant data connected to the project that you are working on. 👍

Gambit142 commented on index.html:
```diff
@@ -216,6 +216,59 @@ <h1>Skills</h1>
+                    adipiscing elit, sed do eiusmod tempor
+                    incididunt ut labore et dolore magna.
+                </p>
+                <form action="https://formspree.io/f/xwkyewre" method="post">
+                    <ul class="form-list">
+                        <li>
+                            <label for="name">Your name</label>
```
- [x] I think it would be nice to give the form a `class` and make it `display: flex`. This way, you would be practicing how to use the flexbox in laying out items as well as fulfilling one of the major project requirements. 👍

Gambit142 commented on styles/style-main.css:
```diff
@@ -154,6 +170,20 @@ h1 {
+
+.contact-me-form li {
+  list-style: none;
+  border: solid gray;
+  border-radius: 5px;
```
- [x] I think it would be a wonderful idea to make the border of all input fields `1px solid #dfdfe0` and the outlines can have a color of `#1a2236` as seen in the Figma template that you are using. Kindly check below for further clarifications:

## Your Design

![image](https://user-images.githubusercontent.com/77106962/144140303-358cbe04-557b-45fd-97ca-7c765e1d5bf0.png)

## Figma Design 

![image](https://user-images.githubusercontent.com/77106962/144140369-46593f09-ca4b-4e20-af55-4f5f042bc3e0.png)

Gambit142 reviewed (changes requested):
Hi @eduardosancho 👋, 

Good job so far!

## Highlight
- Good pull request title 💯 
- Good pull request summary 💯 
- Nice implementation of form validation 💯 
- Perfect use of `action` and `method` attribute

There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

eduardosancho committed:
Fix suggestions by code-reviewer.

-Change normal image to a background in "contact-me" section.
-Use flexbox to organize the elements of the form.
-Improve the aesthetics of the input areas with outline.

eduardosancho committed:
Add a class to the form and style using flexbox

eduardosancho committed:
Fix linters before doing git push

Gambit142 commented on styles/style-main.css:
```diff
@@ -461,3 +495,83 @@ h1 {
+.contact-me {
+  display: flex;
+  margin-top: 50px;
+  background-image: url(/images/mobile_geometry_1.png);
+  background-repeat: no-repeat;
```
Nice job implementing most changes suggested. You have one thing left out;

- [x] The file path to the background image for this section is broken. I think you can fix it by changing it to 

 ` background-image: url(../images/mobile_geometry_1.png);`

Gambit142 reviewed (changes requested):
Hi @eduardosancho 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there is one more which hasn't been addressed yet.


 ## Required Changes ♻️
_Kindly check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

eduardosancho committed:
Fix typo in contact section background image

carlos-ssh reviewed (approved):
# 🍾🍾 **STATUS APPROVED** 👏🏽👏🏽

Hi @eduardosancho , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



![Approved](https://user-images.githubusercontent.com/22258274/138466108-6b14c541-6a88-44c5-ba76-c64992ac1835.gif)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
